Assignment3/calculateFrequency.js

```diff
@@ -0,0 +1,11 @@
+exports.calculateFrequency = function (string) {
+  const frequency = {};
+  var character;
+  string = string.replace(/\s/g,'');
```

Comment:
modifying parameter which is passed in a function is not a good practice. Use local variable instead.